tests/unit/test-class-wc-payments-features.php

```diff
@@ -72,33 +73,18 @@ function ( $pre_option, $option, $default ) {
 	}
 
 	public function test_is_platform_checkout_is_returned_as_true() {
-		update_option( '_wcpay_feature_platform_checkout', '1' );
-		$this->assertTrue( WC_Payments_Features::is_platform_checkout_enabled() );
+		add_option( WC_Payments_Account::ACCOUNT_OPTION, [ 'account' => [ 'platform_checkout_eligible' => true ] ] );
+		$this->assertTrue( WC_Payments_Features::is_platform_checkout_eligible() );
 	}
 
-	/**
-	 * @dataProvider is_platform_checkout_falsy_value_provider
-	 */
 	public function test_is_platform_checkout_is_returned_as_false_if_not_equal_1() {
```

Comment:
This `test_is_platform_checkout_is_returned_as_false_if_not_equal_1` kinda looks misspelled. Maybe it was meant to be `test_if`... or even `test_is_platform_checkout_ineligible_if_not_equal_1`

Reply:
Updated in 704ab29f5533d1673e631ad089ed54f9276e018d.